README.md

```diff
@@ -33,7 +33,7 @@ with the filename of the changed file. (The symbol may be changed with the
 OPTIONS are given below:
       --all=false:
             Include normally ignored files (VCS and editor special files).
-  -c, --config="":
+  -c, --config="reflex.conf":
```

Comment:
This isn't the output of running `reflex -h`, is it?

You should update the `-c` documentation string to mention `reflex.conf`, particularly because it's not a normal flag default value.

Reply:
I would much appreciate some advice on how best to do that.

We can edit the help string to append the phrase `Default: reflex.conf` at the end, so it appears like so:

```
  -c, --config="":
            A configuration file that describes how to run reflex
            (or '-' to read the configuration from stdin). Default: reflex.conf
```

But that's not how pflag prefers to indicate default arg values, instead this feels better:

```
  -c, --config="reflex.conf":
            A configuration file that describes how to run reflex
            (or '-' to read the configuration from stdin).
```

However, unless I'm missing something, #2 can only be done through the `value` arg of `pflag.StringVarP` call, which I don't want to do because there's a special case, where `-c` is not passed, but `reflex.conf` doesn't exist either. It's not simply that _"default value for -c is reflex.conf"_ but rather _"default value for -c is reflex.conf if the file exists in current working dir"_, and I'm at loss as to how best to handle that third case.

I'm leaning towards a spin on the first method above mentioned, like so:

```
 -c, --config="":
            A configuration file that describes how to run reflex
            (or '-' to read the configuration from stdin).
            Default: reflex.conf – if it exists in current working directory.
```